risc0/zkp/src/prove/accum.rs

```diff
@@ -57,6 +60,7 @@ impl<E: Elem> Accum<E> {
 }
 
 pub struct Handler<'a, F: Field> {
+    // TODO what is p?
```

Comment:
@jbruestle one TODO remaining -- can you clarify the role of `p` in the Handler?

Reply:
@shkoo Do you know what `p` refers to in this context?

Reply:
p is a mutex lock (an implementation detail so it can be used in multiple threads and as a callback) containing a "Accum" structure, that's commented as "Tracks grand product accumulations for PLONK-style permutation arguments." on line 24

Reply:
Thanks for clarifying, @shkoo
I've edited the TODO to read, "p is a mutex lock that contains an Accum structure;" lmk if you'd revise that.